docs/sources/tempo/setup/operator/monitor.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Monitor'
+description: Monitor TempoStack instances
+menuTitle: Monitor
+weight: 300
+aliases:
+- /docs/tempo/operator/monitor
+---
+
+## Distributed Tracing
```

Comment:
The first heading on a page is usually a 1st-level heading (i.e., # Tempo) instead of a sub-heading (## Sub-tempo).

Ideally, this first heading would match the title defined in the front matter (doc-validator will have an error message if they don't match). 

If you were going to describe the purpose of this page, what would you say? (I'm asking because the page title and description don't seem to match with the page contents.) An updated page title and a brief introduction could go a long way to help people understand the benefit of what they will have from following this procedure. 

Also, consider adding a "[Before you begin" section](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/) that describes any prerequisites or assumptions. For example, we use this type of section in the Tempo documentation for procedures. For example: https://grafana.com/docs/tempo/latest/setup/linux/

Reply:
Once I have a better understanding of the purpose of your page, then I can provide more focused feedback.

Reply:
I have added the title and some description

Reply:
Thank you.

Reply:
This looks much better. Thank you!